newIDE/app/src/InstancesEditor/CompactInstancePropertiesEditor/CompactPropertiesSchema.js

```diff
@@ -414,15 +418,83 @@ const getKeepRatioField = ({
   getNextValue: (currentValue: boolean) => !currentValue,
 });
 
+const getOpacityField = ({ i18n }: {| i18n: I18nType |}) => ({
+  name: 'Opacity',
+  getLabel: () => i18n._(t`Opacity`),
+  valueType: 'number',
+  getValue: (instance: gdInitialInstance) => {
+    const opacity = instance.getOpacity();
+    return Math.round((opacity / 255) * 100);
+  },
+  setValue: (instance: gdInitialInstance, newValue: number) => {
+    const newOpacity = Math.round((newValue / 100) * 255);
+    const opacity = Math.max(0, Math.min(255, newOpacity));
+    instance.setOpacity(opacity);
+  },
+  renderLeftIcon: className => <Opacity className={className} />,
+  formatDisplayedValueOnBlur: (value: string): string => {
+    return `${value}%`;
+  },
+  formatDisplayedValueOnFocus: (displayedValue: string): string => {
```

Comment:
These names don't really show that they are inverse of each other.
Maybe something like "getDisplayedValueFromValue" and "getValueFromDisplayedValue"?